circuit_puzzles/programs/oracle_mutation.clsp

```diff
@@ -7,10 +7,8 @@
                 price_updatable_after_seconds
                 price_updatable_threshold_percent
                 price_expiration_seconds
-                announcer_delay
-                announcer_deposit
                 current_timestamp
-                ; price_announcers -> ((inner_puzzle_hash args_hash price))
+                ; price_announcers -> ((inner_puzzle_hash args_hash price deposit))
```

Comment:
price_announcers would need to include delay as well if we allow this to be less than the max defined in Statutes

Reply:
this is invalid comment